apps/fishing-map-e2e/src/e2e/map/eez-download.cy.ts

```diff
@@ -0,0 +1,217 @@
+    waitForMapLoadTiles()
+    scrollSidebar('bottom', 1000)
+    cy.getBySel(SIDEBAR_TOGGLE_EEZ).click()
+    cy.getBySel('map-search-button').click()
+    // I need to add a delay because it doesnt work propetly the autocomplete if we type so fast
+
+    cy.getBySel('map-search-input').click().type(SEARCH_EEZ, { delay: 200 })
+    cy.getBySel('map-search-results').findBySelLike('map-search-result').first().click()
+    getMapCanvas().click('center')
+    cy.intercept(API_URL_GALAPAGOS_INFO).as('areaInfo')
+    cy.getBySel(MAP_POPUP_EEZ_SECTION, { timeout: 10000 })
+      .findBySelLike('download-activity-layers')
+      .click()
+    cy.getBySel('download-activity-byvessel')
+      .findBySelLike('area-name')
+      .contains(SEARCH_EEZ_FULL_NAME)
+
+    // The download button is not available until the data of the area is loaded
+    cy.wait('@areaInfo')
+  })
+
+  afterEach(() => {})
+
+  it('Should download Active vessels for ' + SEARCH_EEZ, () => {
+    verifyFileDownload(zipPath, 'activity-vessel')
+    cy.task('unzipping', { path: getDownloadsFolderPath() + '/', file: zipFilename }).then(() => {
+      verifyCSV([
+        'Time Range',
+        'Vessel Name',
+        'Flag',
+        'Entry Timestamp',
+        'Exit Timestamp',
+        'Gear Type',
+        'Vessel Type',
+        'MMSI',
+        'IMO',
+        'CallSign',
+        'First Transmission Date',
+        'Last Transmission Date',
+        'Apparent Fishing Hours',
+      ])
+    })
+
+    cy.log('DOWNLOAD CSV - GROUP BY MMSI AND MONTH')
+    deleteDownloadsFolder()
+    selectOptions(REPORT_FORMAT_CSV, GROUPBY_MMSI, GROUPBY_MONTH)
+    verifyFileDownload(zipPath, 'activity-vessel')
+    cy.task('unzipping', { path: getDownloadsFolderPath() + '/', file: zipFilename }).then(() => {
+      verifyCSV([
+        'Time Range',
+        'mmsi',
+        'Entry Timestamp',
+        'Exit Timestamp',
+        'Apparent Fishing Hours',
+      ])
+    })
+
+    cy.log('DOWNLOAD CSV - GROUP BY GEAR+FLAG AND DAY')
+    deleteDownloadsFolder()
+    selectOptions(REPORT_FORMAT_CSV, GROUPBY_FLAG_GEAR, GROUPBY_DAY)
+    verifyFileDownload(zipPath, 'activity-vessel')
+    cy.task('unzipping', { path: getDownloadsFolderPath() + '/', file: zipFilename }).then(() => {
+      verifyCSV(['Time Range', 'Flag', 'Geartype', 'Vessel IDs', 'Apparent Fishing Hours'])
+    })
+
+    cy.log('DOWNLOAD JSON - GROUP BY FLAG AND MONTH')
+    deleteDownloadsFolder()
+    selectOptions(REPORT_FORMAT_JSON, GROUPBY_FLAG, GROUPBY_MONTH)
+    verifyFileDownload(jsonPath, 'activity-vessel')
+    verifyJson(['date', 'flag', 'hours', 'vesselIDs'])
+
+    cy.log('DOWNLOAD JSON - GROUP BY GEAR AND DAY')
+    deleteDownloadsFolder()
+    selectOptions(REPORT_FORMAT_JSON, GROUPBY_GEAR, GROUPBY_DAY)
+    verifyFileDownload(jsonPath, 'activity-vessel')
+    verifyJson(['date', 'geartype', 'hours', 'vesselIDs'])
+  })
+
+  it('Should download gridded activity for ' + SEARCH_EEZ, () => {
+    deleteDownloadsFolder()
```

Comment:
Why deleting the downloadsFolder as first step of the test if it is already deleted [beforeEach](https://github.com/GlobalFishingWatch/frontend/pull/2154/files#diff-adc1e64389f81e8dcfdb8819c4468f695ab2ce022cfbb0c03fce50699dd6942bR78) test?